src/components/permissions/routes.ts

```diff
@@ -57,7 +58,17 @@ Authenticates user and checks they belong to requested org (id). Returns:
   - JWT (with orgId included)
 */
 const routeLoginOrg = async (request: any, reply: any) => {
-  const { orgId, sessionId } = request.body
+  const { auth, body } = request
+  if (!body)
+    return reply.send({
+      success: false,
+      message: 'Missing orgId in body.',
+    })
+  if (!body.orgId) return reply.send({ success: false, message: 'orgId not provided' })
```

Comment:
Ok - I'm removing all my changes from this method, this was added while I was testing, and though it would be nice to give some clearer reults for missing properties. I was getting not clear errors before